docusaurus/docs/quickstarts/network/help/change-admin-password.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Change Password
+---
+
+Follow these instructions to change your quickstart admin's password. A random password was generated for you when you ran `expressInstall` and saved in your *.env file as `ZITI_PWD`. After changing the password, also change the value of `ZITI_PWD` in `~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env` to match. This variable is used by the `zitiLogin` function.
+
+## Use the `ziti` CLI {#ziti-cli}
+
+```bash
+$ zitiLogin
```

Comment:
zitiLogin is helpful. do you think we should keep it or start using `ziti edge login` instead? i feel like this whole page should avoid mentioning the quickstart stuff regardless since these are totally ancillary to the quickstart.

Reply:
It does feel like an opportunity to introduce the `ziti` CLI, but the login command is pretty long with the certificate param.

```bash
$ type zitiLogin

zitiLogin is a function
zitiLogin () 
{ 
    "${ZITI_BIN_DIR-}/ziti" edge login "${ZITI_EDGE_CTRL_ADVERTISED}" -u "${ZITI_USER-}" -p "${ZITI_PWD}" -c "${ZITI_PKI_OS_SPECIFIC}/${ZITI_EDGE_CONTROLLER_ROOTCA_NAME}/certs/${ZITI_EDGE_CONTROLLER_INTERMEDIATE_NAME}.cert"
}
```

Even the console stuff is quickstart-specific unless we assume they already know how to login to the console. If we trim the quickstart-specific stuff all that remains is the `ziti edge update authenticator updb -s` which I was going to add to the CLI Basics page anyway.

Reply:
added the change updb hint in https://ziti-doc-git-issue-106-router-docs-overhaul-openziti.vercel.app/operations/cli-basics#change-password